FWCore/Utilities/src/ResolveSymbolicLinks.cc

```diff
@@ -0,0 +1,54 @@
+#include "FWCore/Utilities/interface/ResolveSymbolicLinks.h"
```

Comment:
Please change the file name to also begin with a lower case letter to match the function name.